modules/tinymce/src/plugins/advlist/main/ts/Plugin.ts

```diff
@@ -14,6 +14,9 @@ export default () => {
     if (editor.hasPlugin('lists')) {
       Buttons.register(editor);
       Commands.register(editor);
+    } else {
+      // eslint-disable-next-line
+      console.error('Please use the Lists Plugin together with the AdvList plugin.');
```

Comment:
Might be better to use lowercase here to correlate with the names of the plugins in the editor settings?

Reply:
Following same pattern as mentioned on the ticket here: https://github.com/tinymce/powertable/blob/develop/src/main/ts/Plugin.ts#L39

Just changed from console.log to console.error

Reply:
We use the formal name elsewhere already, so I'm fine with using `Lists`. eg: https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/plugins/paste/main/ts/alien/DetectProPlugin.ts#L14

However, this should not have a capital P in plugin.

- [ ] Replace capital P in plugin

Reply:
Done.